.github/workflows/tests.yml

```diff
@@ -91,5 +91,5 @@ jobs:
     uses: cloudposse/.github/.github/workflows/shared-go-auto-release.yml@main
     with:
       publish: true
-      runs-on: '["runs-on=${{ github.run_id }}", "runner=default"]'
+      runs-on: '["runs-on=${{ github.run_id }}", "runner=medium", "disk=large"]'
```

Comment:
Seems like this is an unnecessary arg that could just be in the shared workflow.